Author: Mathias Behrle <mathiasb@m9s.biz>
Description: Set the data path of the server to the home of the Debian tryton user.
--- tryton-server.orig/etc/trytond.conf	2012-03-28 11:25:15.898250683 +0200
+++ tryton-server/etc/trytond.conf	2012-03-28 11:50:10.295431092 +0200
@@ -57,7 +57,7 @@
 #smtp_user = False
 
 # Configure the path to store attachments and sqlite database
-#data_path = /var/lib/trytond
+data_path = /var/lib/tryton
 
 # Allow to run more than one instance of trytond
 #multi_server = False
